components/ipfs_ui/components/uninstalledView.tsx

```diff
@@ -11,14 +11,29 @@ import { PaddedButton, Section, Title, Error } from '../style'
 
 interface Props {
   daemonStatus: IPFS.DaemonStatus
+  installationProgress: IPFS.InstallationProgress
   onInstall: () => void
 }
 
 export class UninstalledView extends React.Component<Props, {}> {
   constructor (props: Props) {
     super(props)
   }
-
+  progress () {
+    if (!this.props.daemonStatus.installing ||
+        this.props.installationProgress.downloaded_bytes === -1 ||
+        this.props.installationProgress.total_bytes === -1) {
+      return ''
+    }
+    const bytes = this.props.installationProgress.downloaded_bytes
+    const total = this.props.installationProgress.total_bytes
+    const percentages = 100 * bytes / total
```

Comment:
should we check to make sure total is non zero here?

Reply:
oops, done